src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const declarations = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim())
+    .filter(Boolean);
```

Comment:
[CODE STYLE] - When chaining methods, each method should be on a new line with the same indentation to enhance readability. Adjust the chaining of `split`, `map`, and `filter` methods to match the good example provided in the checklist.